css/styles.css

```diff
@@ -4,26 +4,49 @@ body {
 }
 
 header {
-  background-color: purple;
+  background-color: #562b81;
   color: white;
   padding-top: 1rem;
 }
 
+nav {
+  display: flex;
+}
+
+a {
+  color: white;
+  text-decoration: none;
+}
+
+.entries-btn {
+  margin-left: 2rem;
+  padding-top: 15px;
+  font-size: 1rem;
+}
+
+ul {
+  list-style-type: none;
+}
+
+li {
+  margin-bottom: 2rem;
+}
+
 header > h2 {
   height: 2rem;
 }
 
 img {
   width: 100%;
-  height: ;
+  height: 400px;
```

Comment:
I would remove this `height` property. It is better to just set the width and let the height autogenerate